BackOfficeEditorView.Core/Hubs/SyncHub.cs

```diff
@@ -91,12 +95,15 @@ public async Task RemoveViews(object sessionId)
             await Clients.Group(GroupID).SendAsync("ContentViewed", _viewManager.FetchAllViews());
         }
 
-        public async Task GetContentLocksForContentId(object contentIdStr)
+        public async Task GetContentLocksForContentId(object contentIdStr, object culture)
         {
             if (!int.TryParse(contentIdStr?.ToString(), out var contentId))
                 return;
 
-            var contentLocks = _contentLockManager.GetCurrentContentLocks(contentId);
+            var cultureStr = !string.IsNullOrEmpty(culture.ToString()) ? culture.ToString() : null;
```

Comment:
Needs a null check on culture ->
`!string.IsNullOrEmpty(culture?.ToString())`

or simplify the assignment to `var cultureStr = culture?.ToString();`

Reply:
Thank you, I've pushed an update on that line.